azkaban-common/src/main/java/azkaban/scheduler/TriggerBasedScheduleLoader.java

```diff
@@ -44,7 +44,7 @@
   private long lastUpdateTime = -1;
 
   public TriggerBasedScheduleLoader(final TriggerManager triggerManager,
-                                    final String triggerSource) {
+      final String triggerSource) {
```

Comment:
This looks like a formatting change.  Is it intentional? I though the earlier code was formatted properly by the save action.